Makefile

```diff
@@ -58,13 +75,21 @@ really-clean: clean
 deploy:
 	@./scripts/deploy.sh ${BROKER_IMAGE}:${TAG} ${REGISTRY} ${ORG}
 
-test:
-	go test ./pkg/...
-
 cleanup-ci:
 	./scripts/broker-ci/cleanup-ci.sh
 
-ci:
+ci: fmtcheck vet lint build test ## Run CI (fmtcheck, vet, lint, build, test)
```

Comment:
```make ci``` is meant to be both a command run by travis and locally. When run locally, it will do a quick sanity check to test your patch works.  I like the idea about moving these into the makefile, but I think they shouldn't be apart of ```make ci```. How about ```make travis```?